modules/qtx_modules_handler.php

```diff
@@ -89,8 +102,13 @@ public static function get_modules_defs() {
                 'name'         => 'Yoast SEO',
                 'plugin'       => 'wordpress-seo/wp-seo.php',
                 'incompatible' => 'wp-seo-qtranslate-x/wordpress-seo-qtranslate-x.php'
+            ),
+            array(
+                'id'           => 'slugs',
+                'name'         => 'Slugs',
+                'plugin'       => true,
```

Comment:
We should rather enable that if the user checks "enable experimental slugs", so the other way around.
But I see why it is easier like this for now. The module has still the legacy code of the plugin, so the init sequence should be refactored.